tests/functional/LoaderTest.php

```diff
@@ -67,4 +70,11 @@ public function testLoad()
 
         $this->assertFileEquals(__DIR__.'/data/users.jsonld', 'vfs://output.jsonld');
     }
+
+    public function pipelineRunner(): PipelineRunnerInterface
+    {
+        return new \Kiboko\Component\Pipeline\PipelineRunner(
+            new NullLogger()
+        );
+    }
```

Comment:
Use a copy of this [PipelineRunner](https://github.com/php-etl/csv-flow/blob/master/tests/functional/PipelineRunner.php)